gammapy/obs/observation.py

```diff
@@ -128,6 +141,8 @@ def select_range(self, selection_variable, value_range, inverted=False):
 
         # build and apply mask
         mask = (value_range[0] <= value) & (value < value_range[1])
+        if np.allclose(value_range[0], value_range[1]):
+            mask = value_range[0] == value
```

Comment:
Ok, but in a later comment you requested to avoid superfluous parentheses :-)
Done.